trxdb-loader/loader.go

```diff
@@ -73,6 +76,8 @@ func NewTrxDBLoader(
 		parallelFileDownloadCount: parallelFileDownloadCount,
 		blockFilter:               blockFilter,
 		truncationWindow:          truncationWindow,
+		lastBlockSecs:             make([]float64, 100),
+		lastBlockSecsPointer:      0,
```

Comment:
0 is default value, no need to set it here (we don,t set lastTickBlock or lastTickTime either, let's be consistent)